Add execution instructions (#639)
bumble-tech/appyx: documentation/navigation/multiplatform.md

In the example mentioned earlier, we create a Composable component `BackButton` that includes an `ArrowBack` icon. 
When this button is clicked, it triggers the back event through the `backEvents` `Channel`. 

## Setting up the environment for execution

Setting up the environment for execution

> > **Warning**
> > For running the iOS target, you need a Mac with macOS to write and run iOS-specific code on simulated or real devices.
> > This is an Apple requirement.
> >
> > The instructions here are tweaked and inspired from the [Compose-Multiplatform-iOS template](https://github.com/JetBrains/compose-multiplatform-ios-android-template).
> > To work with this project, you need the following:

Reviewer: ```suggestion
{==

**Warning**

In order to launch the iOS target, you need a Mac with macOS to write and run iOS-specific code on simulated or real devices.

This is an Apple requirement.

The instructions here are tweaked and inspired from the [Compose-Multiplatform-iOS template](https://github.com/JetBrains/compose-multiplatform-ios-android-template).

==}

To work with this project, you need the following:
```
